packages/clay-toolbar/stories/index.tsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import '@clayui/css/lib/css/atlas.css';
+import {storiesOf} from '@storybook/react';
+import React from 'react';
+
+import ClayToolbar from '../src';
+
+storiesOf('Components|ClayToolbar', module)
+	.add('default', () => (
```

Comment:
Maybe for the default example https://pat270.github.io/clay3-test-site/vclay-table-dd/content/tbar/#tbar might be better?